palantir/docker/publish-docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+IMAGE_NAME=palantirtechnologies/circle-hadoop-2
+
+echo $PWD
+cd palantir/docker
+echo $PWD
+docker build . -t ${IMAGE_NAME}
```

Comment:
do you want to only ever push `latest`, or do you also want to include a versioned tag here?

Reply:
we talked about it, only pushing `latest` is expected.